scripts/crawl_zenodo.py

```diff
@@ -432,8 +445,15 @@ def create_readme(dataset, path):
 
 [![DOI](https://www.zenodo.org/badge/DOI/{1}.svg)](https://doi.org/{1})
 
-Crawled from Zenodo"""
-                    .format(dataset["title"], dataset["doi_badge"]))
+Crawled from Zenodo
+
+## Description
+
+{2}""".format(
+                dataset["title"],
+                dataset["doi_badge"],
+                html2markdown.convert(dataset["description"]).replace("\n", "<br />"))
```

Comment:
Not all descriptions will be HTML, is there a way to check that, or do you confirm that it's harmless to convert from HTML if the description is just plain text?

Reply:
Yes I tested it and if the input to convert is regular string, it will just return the string as is